### PR TITLE
CB-2647 top bar menu redesign

### PR DESCRIPTION
**Pull request by Wroud**

### CODE REVIEW

File: webapp/packages/plugin-log-viewer/src/LocaleService.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/*
```
Comment (devnaumov) [inline webapp/packages/plugin-log-viewer/src/LocaleService.ts:1]: Please change licence text

File: webapp/packages/plugin-sql-editor-navigation-tab/src/LocaleService.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
+/*
```
Comment (devnaumov) [inline webapp/packages/plugin-sql-editor-navigation-tab/src/LocaleService.ts:1]: please change the licence text

File: webapp/packages/plugin-tools-panel/src/LocaleService.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
+/*
```
Comment (devnaumov) [inline webapp/packages/plugin-tools-panel/src/LocaleService.ts:1]: please change license text

File: webapp/packages/plugin-top-app-bar/src/TopNavBar/shared/topMenuStyles.ts [hunk 4]
```diff
@@ -18,13 +18,37 @@ export const topMenuStyles = css`
   menu-box {
     composes: theme-text-on-primary from global;
     background-color: #338ecc!important;
+    clip-path: inset(0px -16px -16px -16px);
```
Comment (devnaumov) [inline webapp/packages/plugin-top-app-bar/src/TopNavBar/shared/topMenuStyles.ts]: please leave a comment that explains what it does

**Review by devnaumov — CHANGES_REQUESTED** [review]

File: webapp/packages/plugin-log-viewer/src/LocaleService.ts (hunk 1 above)
Comment (Wroud) [reply webapp/packages/plugin-log-viewer/src/LocaleService.ts:1]: fixed

File: webapp/packages/plugin-sql-editor-navigation-tab/src/LocaleService.ts (hunk 2 above)
Comment (Wroud) [reply webapp/packages/plugin-sql-editor-navigation-tab/src/LocaleService.ts:1]: fixed

File: webapp/packages/plugin-tools-panel/src/LocaleService.ts (hunk 3 above)
Comment (Wroud) [reply webapp/packages/plugin-tools-panel/src/LocaleService.ts:1]: fixed

File: webapp/packages/plugin-top-app-bar/src/TopNavBar/shared/topMenuStyles.ts (hunk 4 above)
Comment (Wroud) [reply webapp/packages/plugin-top-app-bar/src/TopNavBar/shared/topMenuStyles.ts]: fixed

**Review by devnaumov — APPROVED** [review]

**Pull request merged**